client/src/components/Q&A/QA.jsx

```diff
@@ -96,19 +99,23 @@ const QuestionsAndAnswers = () => {
         setAllQuestionsData(data);
         setQuestionsData(container);
       })
-      // .then(() => setCookie(document.cookie))
+      .then(() => {
+        localStorage.setItem(`${document.cookie}`, document.cookie);
```

Comment:
How are you planning to use localStorage in the future?

Reply:
@joshgarza Yes. I'm planning to store the cookie in the local storage so it's easier to check.